.well-known/security.txt

```diff
@@ -0,0 +1,9 @@
+# Kam nahlásit bezpečnostní chybu / Where to report security issue
```

Comment:
To vypadá na změny z jiných commitů. Nejspíš se nepodařil rebase. To by tu rozhodně být nemělo